### PR TITLE
Add delay to recorder

### LINKED ISSUE #312: Allow a delay before actually recording
This is against recording mouse clicks when starting the recording. As discussed offline: it's good SDK functionality to provide as a setting or perhaps even default of 100ms delay.

### PR DESCRIPTION
**Pull request by joggienl**
Add delay to actually start recording. This is in favor of using the backend his/her trim feature.

Closes #311 
Fixes #312

### CODE REVIEW

File: src/audio/audio-recorder.js [hunk 1]
```diff
@@ -258,20 +261,25 @@ export default class AudioRecorder {
       throw new Error('Already recording, stop recording first.');
     }
 
-    this.audioContext.resume();
+    const delay = disableDelay ? 0 : 100;
```
Comment (kvdb) [inline src/audio/audio-recorder.js]: i'd rename parameter to: recordingDelay=100 by default, override to 0 or 250 or whatever you want by sdk user.

**Comment by kvdb** [comment]
Patch is fine, got one suggestion you may choose to implement.

File: src/audio/audio-recorder.js (hunk 1 above)
Comment (joggienl) [reply src/audio/audio-recorder.js]: to be honest, I don't think we should allow to override this value (at this point). Maybe our future recorder could provide also a way to override the value specifically but at this point I made the choice to either allow the default delay (or not).

Comment (kvdb) [reply src/audio/audio-recorder.js]: I don't have very strong opinions here. so. fine, i'll merge as is.

**Pull request merged**